ledger/participant-integration-api/src/main/scala/platform/store/backend/UpdateToDBDTOV1.scala

```diff
@@ -139,16 +139,20 @@ object UpdateToDBDTOV1 {
         )
 
       case u: Update.TransactionAccepted =>
-        // TODO append-only: add support for Daml Exceptions / Rollback nodes (Story for this: https://digitalasset.atlassian.net/browse/DPP-374)
+        // TODO append-only:
```

Comment:
Not quite sure if you want to keep that todo and in what form. With sadbonx classic you do have the option of relying on conformance tests now. Of course unit tests can still be sensible.

Reply:
We anyway wanted to have extensive unit testing for this guy. The core of the idea of composing indexer code like this was to evacuate business logic out of jdbc/akka-stream/future/dao maze.
this is the business logic - pure function - ideal for unit testing with high coverage